src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-                bzero(data, INSTANCE_DATA_LENGTH + 1);
-                read(connFd, data, INSTANCE_DATA_LENGTH);
-                string graphId = (data);
-                graphId = Utils::trim_copy(graphId, " \f\n\r\t\v");
-                instance_logger.log("Received Graph ID: " + graphId, "info");
+    for (localGraphMapIterator = localGraphMap.begin(); localGraphMapIterator != localGraphMap.end();
+         ++localGraphMapIterator) {
+        long startVid = localGraphMapIterator->first;
 
-                write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-                bzero(data, INSTANCE_DATA_LENGTH + 1);
-                read(connFd, data, INSTANCE_DATA_LENGTH);
-                string partitionId = (data);
-                partitionId = Utils::trim_copy(partitionId, " \f\n\r\t\v");
-                instance_logger.log("Received Partition ID: " + partitionId, "info");
+        adjacencyIndex[counter] = startVid;
+        counter++;
+    }
 
-                string aggregateLocation =
-                    Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.aggregatefolder");
-                string fileName = graphId + "_centralstore_" + partitionId;
-                string fullFilePath = aggregateLocation + "/" + fileName;
-                string result = "false";
+    adjacencyIndex[partitionVertexCount] = -1;
 
-                bool fileExists = Utils::fileExists(fullFilePath);
+    long entireGraphSize = atol(graphVertexCount.c_str());
+    float mu = (damp / entireGraphSize);
+    unordered_map<float, float> resultTreeMap;
+    // calculating local pagerank
+    map<long, double> rankMap;
 
-                if (fileExists) {
-                    result = "true";
-                }
+    map<long, long> inDegreeDistribution;
 
-                write(connFd, result.c_str(), result.size());
-                instance_logger.log("Sent : " + result, "info");
-            } else if (fileType.compare(JasmineGraphInstanceProtocol::FILE_TYPE_CENTRALSTORE_COMPOSITE) == 0) {
-                write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-                bzero(data, INSTANCE_DATA_LENGTH + 1);
-                read(connFd, data, INSTANCE_DATA_LENGTH);
-                string fileName = (data);
-                fileName = Utils::trim_copy(fileName, " \f\n\r\t\v");
-                instance_logger.log("Received File name: " + fileName, "info");
+    std::string aggregatorFilePath = Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder");
+    std::string iddFilePath = aggregatorFilePath + "/" + graphID + +"_idd_" + partitionID;
```

Comment:
```suggestion
    std::string iddFilePath = aggregatorFilePath + "/" + graphID + "_idd_" + partitionID;
```

Reply:
All such occurrences of `+ +` are replaced in next PR

Reply:
Ok